test/powerflow.jl

```diff
@@ -37,4 +37,25 @@ import ExaPF: PowerSystem
             @test convergence.n_iterations == 2
         end
     end
+    @testset "Deport only powerflow on device $device" for device in DEVICES
```

Comment:
Not sure we want to test all devices there, but I have to think more about that.

Reply:
Until it breaks... why not? In the end we want to transfer from any device to any device if needed.